WordPress/metadata/release_notes.txt

```diff
@@ -1,3 +1 @@
 Locked and loaded—there’s now a lock icon for disabled cells in the block editor so you know when they aren’t in use.
-
-We also updated the placeholder text for VideoPress videos that don’t belong to your site. (Hopefully they don’t feel left out.)
```

Comment:
Thank you @SiobhyB for pointing out that the VideoPress update only applies to Jetpack 👍

Reply:
Thank you for quickly addressing @mokagio!